pkg/broker/ingress/options.go

```diff
@@ -20,61 +20,53 @@ import (
 	"context"
 
 	"cloud.google.com/go/pubsub"
-	cloudevents "github.com/cloudevents/sdk-go/v2"
-	"github.com/google/knative-gcp/pkg/broker/config"
+	cev2 "github.com/cloudevents/sdk-go/v2"
+	cepubsub "github.com/cloudevents/sdk-go/v2/protocol/pubsub"
+	"github.com/google/wire"
 	"knative.dev/eventing/pkg/kncloudevents"
 )
 
-// HandlerOption is the option to configure ingress handler.
-type HandlerOption func(*handler) error
+// HandlerSet provides a handler with a real HTTPMessageReceiver and pubsub MultiTopicDecoupleSink.
+var HandlerSet wire.ProviderSet = wire.NewSet(
```

Comment:
Let's move this to handler.go or create a provider.go?

Reply:
Moved to handler.go